_posts/2016/2016-12-20-cqrs-and-elasticsearch.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "CQRS and Elasticsearch"
```

Comment:
_"Harnessing the best features of RDBMS and Elasticsearch with CQRS"_ gives a headline-analyzer ranking of 67, which is higher than the other alternatives. What should be the filename of the *.md file, that directly influences the blog post URL? `harnessing-the-best-features-of-rdbms-and-elasticsearch-with-cqrs.html`?